src/test/java/uk/gov/companieshouse/efs/api/paymentreports/service/PaymentReportServiceImplTest.java

```diff
@@ -209,6 +252,7 @@ void sendFinancePaymentFailureReport() throws IOException {
 
         final List<PaymentReportEmailModel> values = emailCaptor.getAllValues();
 
+        assertThat(values.get(0).getHasNoPaymentTransactions(), is(false));
         assertThat(values.get(0).getFileLink(), is(successFileLink));
         assertThat(values.get(0).getFileName(), is(sh19ReportName.replace(".csv", "")));
         assertThat(values.get(1).getFileLink(), is(failedFileLink));
```

Comment:
For completeness add
`assertThat(values.get(1).getHasNoPaymentTransactions(), is(false));`

Reply:
ah yes, added.